docs/src/main/tut/contravariant.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title:  "Contravariant"
+section: "typeclasses"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/functor/Contravariant.scala"
+scaladoc: "#cats.functor.Contravariant"
+---
+# Contravariant
+
+The `Contravariant` type class is for functors that define a `contramap`
+function with the following type:
+
+```scala
+def contramap[A, B](f: B => A): F[A] => F[B]
+```
+
+It looks like regular (also called `Covariant`) [`Functor`](functor.html)'s `map`,
+but with `f` transformation reversed.
+
+Generally speaking, if you have some context `F[A]` for type `A`,
+and you can get `A` value out of `B` value — `Contravariant` allows you to get the `F[B]` context for `B`.
+
+Examples of `Contravariant` instances are [`Show`](show.html) and `scala.math.Ordering` (along with `algebra.Order`).
+
+## Contravariant instance for Show.
+
+Say we have class `Money` with a `Show` instance, and `Salary` class. 
+
+```tut:silent
+import cats._
+import cats.implicits._
+
+case class Money(amount: Int)
+case class Salary(size: Money)
+
+implicit val showMoney: Show[Money] = Show.show(m => s"${m.amount}$$")
+```
+
+If we want to show a `Salary` instance, we can just convert it to a `Money` instance and show it instead.
```

Comment:
I was fully expecting a Jerry Maguire quote to make it's way in here. Have I shown my age? :)